AccountingService/Services/ExchangeRateService.cs

```diff
@@ -30,7 +30,7 @@ private static void ReloadExchangeRates()
 
             var request = new RestRequest();
             request.Method = Method.Get;
-            request.AddHeader("apikey", "[API-KEY]");
+            request.AddHeader("apikey", "6IRdNpTdHNvBw5bd4ouVVoAgo5WpZA2E");
```

Comment:
Please remove this API key. We don't want to leak secrets!

Reply:
done!